singleton.md

```diff
@@ -0,0 +1,27 @@
+#Singleton-Pattern
```

Comment:
Bitte leerzeichen zwischen ## und der überschrift einfügen. In der aktuellen README.md ist auch ein markdown linter verlinkt.